dcos/mesos.py

```diff
@@ -567,7 +584,9 @@ def _framework_dicts(self, inactive=False, completed=False):
                 yield framework
 
         for framework in self.state()['frameworks']:
-            if inactive or framework['active']:
+            if framework['active'] and active:
```

Comment:
what are the other state possibilities here a comment would be great, also couldn't this two conditions fit in one line?